BinaryObjectScanner/Protection/SecuROM.cs

```diff
@@ -346,134 +435,49 @@ private static string GetV8WhiteLabelVersion(PortableExecutable exe)
                 return "SecuROM Matroschka Package";
 
-            // TODO: I know this reads like debug output, but this should never happen, so I think I'd always want it there?
-            if (matroschka.Entries == null)
+            if (package.Entries == null || package.Entries.Length == 0)
                 return "SecuROM Matroschka Package - No Entries? Please report"; 
                 
-            var entry = matroschka.Entries[1];
+            // The second entry in a Release Control matroschka package is always the encrypted executable
+            var entry = package.Entries[1]; 
             
-            // TODO: I know this reads like debug output, but this should never happen, so I think I'd always want it there?
-            if (entry.MD5 == null)
+            if (entry.MD5 == null || entry.MD5.Length == 0)
                 return "SecuROM Matroschka Package - No MD5? Please report"; 
 
             string md5String = BitConverter.ToString(entry.MD5!);
             md5String = md5String.ToUpperInvariant().Replace("-", string.Empty);
             
-            // Check if encrypted executable is known via hash
-            string gameName;
-            var fileData = matroschka.ReadFileData(entry, includeDebug); // Not used yet, but will be in the future
+            var fileData = package.ReadFileData(entry, includeDebug); // Not used yet, but will be in the future
 
-            if (MatroschkaHashDictionary.TryGetValue(md5String, out gameName))
+            // Check if encrypted executable is known via hash
+            if (MatroschkaHashDictionary.ContainsKey(md5String))
             {
                 // Returning "SecuROM Matroschka Package" technically redundant since implied.
                 // Since non-debug is more common, return first
                 if (!includeDebug)
                     return "SecuROM Release Control"; 
                 
                 // TODO: I'd like to have this debug output be the normal output, but I assume I'm not allowed to do that
-                return $"SecuROM Release Control -  {gameName}";
+                return $"SecuROM Release Control -  {MatroschkaHashDictionary[md5String]}";
             }
             
             // If not known, check if encrypted executable is likely an alt signing of a known executable
             // Filetime could be checked here, but if it was signed at a different time, the time will vary anyways
             var readPathBytes = entry.Path;
-            if (readPathBytes == null)
-                readPathBytes = [];
-
-            string? pathName;
-            if (MatroschkaSizeFilenameDictionary.TryGetValue(entry.Size, out pathName) && pathName == Encoding.ASCII.GetString(readPathBytes).TrimEnd('\0'))
+            if (readPathBytes == null || readPathBytes.Length == 0)
+                return $"SecuROM Release Control - Unknown executable {md5String},{entry.Size}, PLEASE REPORT ON GITHUB IMMEDIATELY!!!";
```

Comment:
Normal "please report" text is fine

Reply:
I know how you're going to feel about this, so if the answer is no, I understand. The average user is probably going to ignore those, I assume moderators have been ignoring those assuming those even make it through the MPF filter, even I also ignored those. Having it in all caps with multiple exclamation points will improve the chances that someone will report this, and it is time-sensitive, so I'd prefer that.
If the answer is no, then just reply with no and I'll change it.

Reply:
The normal text is sufficient. Those users who choose to report things will do so whether it appears to be urgent or not. Yes, they are ignored by moderators because it has no impact on Redump as a project. It is fully up to the users to see what their scans report and then have the motivation to do something about it.

Reply:
>  Those users who choose to report things will do so whether it appears to be urgent or not.
Having been one of those users, I disagree. Regardless, I'll change it back. Can I at least keep the immediately? Maybe the exclamation points if you're willing. The only reason I'm being like this is because unlike most things that need reporting, this is time-sensitive and will stop being possible in the future.